code/modules/cargo/bounties/assistant.dm

```diff
@@ -0,0 +1,238 @@
+/*
+/datum/bounty/item/assistant/strange_object
+	name = "Strange Object"
+	description = "Nanotrasen has taken an interest in strange objects. Find one in maint, and ship it off to CentCom right away."
+	reward = 1200
+	wanted_types = list(/obj/item/relic)
+
+/datum/bounty/item/assistant/scooter
+	name = "Scooter"
+	description = "Nanotrasen has determined walking to be wasteful. Ship a scooter to CentCom to speed operations up."
+	reward = 1080 // the mat hoffman
+	wanted_types = list(/obj/vehicle/ridden/scooter)
+	include_subtypes = FALSE
+
+/datum/bounty/item/assistant/skateboard
+	name = "Skateboard"
+	description = "Nanotrasen has determined walking to be a wasteful. Ship a skateboard to CentCom to speed operations up."
+	reward = 900 // the tony hawk
+	wanted_types = list(/obj/vehicle/ridden/scooter/skateboard)
+*/
+/datum/bounty/item/assistant/stunprod
+	name = "Stunprod"
+	description = "CentCom demands a stunprod to use against dissidents. Craft one, then ship it."
+	reward = 1300
+	wanted_types = list(/obj/item/weapon/melee/baton/cattleprod)
+
+/datum/bounty/item/assistant/soap
+	name = "Soap"
+	description = "Soap has gone missing from CentCom's bathrooms and nobody knows who took it. Replace it and be the hero CentCom needs."
+	reward = 2000
+	required_count = 3
+	wanted_types = list(/obj/item/weapon/soap)
+
+/datum/bounty/item/assistant/spear
+	name = "Spears"
+	description = "CentCom's security forces are going through budget cuts. You will be paid if you ship a set of spears."
+	reward = 2000
+	required_count = 5
+	wanted_types = list(/obj/item/weapon/material/twohanded/spear)
+
+/datum/bounty/item/assistant/toolbox
+	name = "Toolboxes"
+	description = "There's an absence of robustness at Central Command. Hurry up and ship some toolboxes as a solution."
+	reward = 2000
+	required_count = 6
+	wanted_types = list(/obj/item/weapon/storage/toolbox)
+
+/datum/bounty/item/assistant/statue
+	name = "Statue"
+	description = "Central Command would like to commision an artsy statue for the lobby. Ship one out, when possible."
+	reward = 2000
+	wanted_types = list(/obj/structure/sculpting_block)
+
+/datum/bounty/item/assistant/statue/applies_to(obj/O)
+	if(!..())
+		return FALSE
+	var/obj/structure/sculpting_block/S = O
+	if(S && S.sculpted)
+		return TRUE
+	return FALSE
+
+/datum/bounty/item/assistant/cheesiehonkers
+	name = "Cheesie Honkers"
+	description = "Apparently the company that makes Cheesie Honkers is going out of business soon. CentCom wants to stock up before it happens!"
```

Comment:
I should probably be posting this in a feedback thread, but you should ensure that the items themselves can be able to be produced by crewmembers, and again, that the reasoning needs to be logical.